entity-management-solr/src/main/java/eu/europeana/entitymanagement/solr/model/SolrEntity.java

```diff
@@ -0,0 +1,142 @@
+package eu.europeana.entitymanagement.solr.model;
+
+import eu.europeana.entitymanagement.definitions.model.Entity;
+import eu.europeana.entitymanagement.vocabulary.AgentSolrFields;
```

Comment:
Please remove this import and use the constants from EntitySolrFields class instead

Reply:
Done.